CSP_Minizinc_Code.py

```diff
@@ -37,6 +38,12 @@ def __init__(self, active_safety_monitor = SafetyMonitor.NO_SELECTION, active_se
         self.__safety_monitor_platform = safety_monitor_platform
         self.__current_context = current_context
         self.__fixed_deployment = fixed_deployment
+        self.name = 1
+        self.min_force_sensor_count = 1
+        self.min_tactile_sensor_count = 100
+        self.min_memory_fused = 400
+        self.np = 4

```

Comment:
Shall we better name this as n_platforms? If yes, then the same change has to be made in all usages